wicket/src/lib.rs

```diff
@@ -162,13 +162,15 @@ impl Wizard {
     }
 
     fn mainloop(&mut self) -> anyhow::Result<()> {
+        let rect = self.terminal.get_frame().size();
         // Size the rack for the initial draw
         self.state
             .rack_state
             .resize(&self.terminal.get_frame().size(), &MARGIN);
```

Comment:
Could this now be 

```suggestion
            .resize(&rect, &MARGIN);
```

? General question - why does `rack_state.resize()` take a rectangle but `screen.resize()` takes separate width/height args?

Reply:
Good catch!

As for the interfaces, this is just a historical artifact. I should probably pick one or the other, and I think taking height, width params makes the most sense, especially since that's what the terminal gives us when the size changes.

I'll make both these changes in a follow up PR.